Resources/Prototypes/Entities/Markers/Spawners/Random/Decals/dirt.yml

```
@@ -44,6 +44,7 @@
     - FloorJungleAstroGrass
     - FloorAstroIce
     - FloorAstroSnow
+    - FloorMowedAstroSnow # Delta-V
```

nit: Just label as `# DeltaV`


```suggestion
    - FloorMowedAstroSnow # DeltaV
```